File: app/src/main/java/com/goran/quranipiroz/interfaceUtils/readerIndex/FragReaderIndexCallback.java
/*
 * (c) Faisal Khan. Created on 19/2/2022.
 */

package com.goran.quranipiroz.interfaceUtils.readerIndex;

import android.content.Context;

public interface FragReaderIndexCallback {
    void scrollToTop(boolean smooth);

    void sort(Context ctx);
}

File: app/src/main/java/com/goran/quran/karim/dang/interfaceUtils/readerIndex/FragReaderIndexCallback.kt
/*
 * (c) Faisal Khan. Created on 19/2/2022.
 */
package com.goran.quran.karim.dang.interfaceUtils.readerIndex

import android.content.Context

interface FragReaderIndexCallback {
    fun scrollToTop(smooth: Boolean)
    fun sort(ctx: Context)
}